PSvCloud/Public/Edge/Get-CIEdgeSecurityCheck.ps1

```diff
@@ -0,0 +1,157 @@
+function Get-CIEdgeSecurityCheck {
+    <#
+    .SYNOPSIS
+    Retrieves basic security information for a vShield edge
+
+    .DESCRIPTION
+    Retrieves basic vShield edge security information including:
+    - FW enabled (True/False)
+    - FW default action (Allow/Drop)
+    - Any insecure FW rules 
+
+    .PARAMETER Name
+    Specifies the name of the vShield Edge you want to retrieve.
+
+    .PARAMETER CIEdge
+    Specifies the PSCustomObject output of Get-CIEdge.
+
+    .INPUTS
+    System.String
```

Comment:
if you accept a CIEdge object from the pipeline, shouldn't this be **System.Management.Automation.PSCustomObject** instead of **System.String**?